portality/templates/doaj/api_docs.html

```diff
@@ -0,0 +1,95 @@
+{% extends "base.html" %}
+
+{% block extra_stylesheets %}
+<!-- get Swagger UI css -->
+    <link href='/static/portality/vendor/swagger-ui/src/main/html/css/typography.css' media='screen' rel='stylesheet' type='text/css'/>
+    <link href='/static/portality/vendor/swagger-ui/src/main/html/css/reset.css' media='screen' rel='stylesheet' type='text/css'/>
+    <link href='/static/portality/vendor/swagger-ui/src/main/html/css/screen.css' media='screen' rel='stylesheet' type='text/css'/>
+    <link href='/static/portality/vendor/swagger-ui/src/main/html/css/reset.css' media='print' rel='stylesheet' type='text/css'/>
+    <link href='/static/portality/vendor/swagger-ui/src/main/html/css/print.css' media='print' rel='stylesheet' type='text/css'/>
+    <link href='/static/doaj/css/doaj_api.css' media='screen' rel='stylesheet' type='text/css'/>
+
+{% endblock %}
+
+{% block extra_head %}
```

Comment:
It'd be nice to include these in the ```extra_js_bottom``` block instead and see if they still work? Literally changing the name of the block from extra_head to extra_js_bottom should work for that. It will put the scripts at the bottom of the page instead of in the head, allowing browsers to start displaying the page and then render the content on. It makes the biggest difference on slow or mobile (synonymous?) connections.